### PR TITLE
Add CI job

### PR DESCRIPTION
**Pull request by traversaro**
I am adding a CI so that I can then fix https://github.com/robotology/yarp-device-xsensmt/issues/30 .

### CODE REVIEW

**Comment by traversaro** [comment]
Indeed it is skipped also in the superbuild: https://github.com/robotology/robotology-superbuild/blob/824c43eb9008bccdf26f471f140e219d1aa97483/cmake/RobotologySuperbuildLogic.cmake#L113 .

File: .github/workflows/conda-forge-ci.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - cron:  '0 2 * * *'
+
+jobs:
+  build:
+    name: '[${{ matrix.os }}@${{ matrix.build_type }}@conda]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build_type: [Release]
+        os: [ubuntu-latest, windows-2019]
```
Comment (RiccardoGrieco) [inline .github/workflows/conda-forge-ci.yml:21]: Why `windows-2019` and not `windows-latest`?

Comment (RiccardoGrieco) [inline .github/workflows/conda-forge-ci.yml:21]: Why `windows-2019` and not `windows-latest`?

Comment (traversaro) [reply .github/workflows/conda-forge-ci.yml:21]: To test against Visual Studio 2019. `windows-latest` only has Visual Studio 2022, and robotology-superbuild still uses Visual Studio 2019. The vast majority of problems on Windows is using something that is not supported on older versions, either for bugs that are fixed in modern versions or due using some new features that are implemented in newer versions. For this reason, I prefer to test VS2019.

Comment (traversaro) [reply .github/workflows/conda-forge-ci.yml:21]: See https://github.com/robotology/yarp-device-xsensmt/pull/31/files#r868948384 .

Comment (RiccardoGrieco) [reply .github/workflows/conda-forge-ci.yml:21]: Got it, thanks!

**Review by RiccardoGrieco — APPROVED** [review]

**Pull request merged**